sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -321,13 +315,13 @@ object DateTimeUtils {
     }
   }
 
-  def instantToMicros(instant: Instant): Long = {
+  def instantToMicros(instant: Instant): SQLTimestamp = {
```

Comment:
This looks not consistent to me. `instantToMicros` indicates it returns number of microseconds and using `Long` looks reasonable.

Reply:
Examples: `def localDateToDays(localDate: LocalDate): Int`, `def daysToLocalDate(days: Int): LocalDate`

Reply:
BTW I don't like the type alias `SQLTimestamp` and `SQLDate`. It just costs me more time when reading code. Also cc @HyukjinKwon @viirya @maropu

Reply:
Ah, I see. What's the original reason to add this alias? Centainly, I feel this kind of cosmetic aliases make the code hard to read somewhat. Any benefit at the cost of at the expense of code readability?

Reply:
The type aliases are mostly used in DataTimeUtils. Inside of this object, we know definitely what's timestamps and dates, and which primitive types represent them.

If you don't mind, I will remove the type aliases at all from DateTimeUtils.

Reply:
Yea, the removal looks fine to me.

Reply:
+1 to remove

Reply:
+1 I remember few times I need to go back to check what it means.